packages/css/src/plugins/atomicify-rules.ts

```diff
@@ -8,8 +8,21 @@ interface PluginOpts {
   selectors?: string[];
   atRule?: string;
   parentNode?: Container;
+  classHashPrefix?: string;
 }
 
+/**
+ * Returns true if a given string is a valid CSS identifier
+ *
+ * @param value the value to test
+ * @returns `true` if given value is valid, `false` if not
+ *
+ */
+const isCssIdentifierValid = (value: string): boolean => {
+  const validCssIdentifierRegex = /^[a-zA-Z\-_]+[a-zA-Z\-_0-9]*$/;
+  return Boolean(value.match(validCssIdentifierRegex));
```

Comment:
oops sorry i just realised, can this be simplified to `validCssIdentifierRegex.test(value)` instead?